# Overview of DNA Structure, Gene Structure and Organization

## DNA and Genes

* DNA
  * Information
    * Store: for cellular function via RNA-protein
    * Reproduce: for offspring via replication
  * Basic unit: 2 complementary polynucleotide chain
    * Generally 4 types (2 pairs) of nucleotides
    * Nitrogen containing base + sugar → Nucleoside
    * Nucleoside phosphate(s) → Nucleotide
      * Nucleoside (NxP) / Deoxyribonucleoside (dNxP)
      * Monophosphate (xMP) / Diphosphate (xDP) / Triphosphate (xTP)
    * Nucleotide: Phosphodiester (covalent) bonding
    * Strands: Hydrogen bonding
  * Double Helix
    * Anti-parallel from 5' (Phosphate) to 3' (Hydroxy)
    * 10bp per turn
    * $\ce{A=T;G#C}$ (replace $\ce{T}$ with $\ce{U}$ for RNA)
    * Maintaining; Assist Replication
* (Human) Genome
  * Genes can be on either strand of DNA
    * 24000 genes for human
    * Only some (30%) genes are expressed at a time (per cell)
  * Arranged in 2 sets (diploid) of 22 autosomes + sex chromosome
    * Each set from each parent
    * 3000 mil. bp. per haploid
  * Chromosome
    * 250 mil. bp. / 2000 genes per chromosome
    * Length approx. $\pu{3 m}$; nucleus size approx. $\pu{10 \micro m}$
    * Numbered from large to small
  * DNA + Protein → Nucleosome
    * Protein: Access Control & Packing
    * Histone + non-histone Proteins
      * 8 Histone core: 2x H2A, 2x H2B, 2x H3, 2x H4
    * N-terminal of histone can be modified to restructure chromosome
  * Packed Chain of Nucleosome → Chromatin
    * Condensed Chromosome during mitosis
    * Can relax for transcription / replication
  * After replication before division: 2 sister chromatid per chromosome
  * locus (plu. loci): location of gene on chromosome
  * allele: variant form of a gene
  * Most genomic DNA are non-coding
    * Coding DNA: for protein synthesis
      * Only 1% of genomic DNA (as exon)
      * Codon = 3 bases → 1 AA
    * junk DNA: no biological function
    * Non-coding DNA
      * Transcriptional Regulation (promoter / enchanter / etc)
      * Origin of Replication
      * Centromere / Telomere
      * Non-coding RNA (rRNA / tRNA / miRNA / etc)
  * Mitochondrial DNA
    * ~1700bp; 37genes; circular chromosome
    * Inherited via maternal egg cell
    * up to 2000 mitochondria per cell
    * Mitochondrial DNA polymerase
    * Mutation → Human disease → Mitochondrial transfer as treatment
  * 99.99% similarity among human populations
    * Polymorphism: multiple alleles
      * Generally single nucleotide polymorphism (SNP)
      * Once every 1000 bp → 3mil. SNP
      * Generally at gene, intron, non-coding region
    * Repetitive copy with number variant
      * Microsatellite: Copy 1-6bp; 5-50 times
        * Trinucleotide repeat → May cause genetic disorder
        * Identification in forensic medicine
      * Minisatellite: Copy 10-60 bp; 5-50 times
        * 
           > 
           > 1000 locations in genomes (telomere)
      
      * Copy Number Variant: >1000 bp: gene / chromosome segment
      * Tandem (Same area) / Interspersed (Scattered copies) Repeat
* Epigenetic
  * Heritable
  * Modification on DNA / DNA-associated proteins (histones)
    * Methylation of DNA
    * Acetylation / Methylation / Deacetylation / Demethylation of Histone
      * Modify N-terminal → condensed / relaxed chromatid
  * Cellular Differentiation & Embryonic Development
  * Lyonization: deactivation of one of the X sex chromosome
    * Same amount of X-based product in XX and XO/XY
    * Inactivated for lifetime of cell and its descendant
      * Barr body: condensed inactivated X-chromosome → visible
    * Stripy XX → Calico cat

## DNA Replication and Maintainence

* Cell Cycle: ~ 24 hrs.
  * Interphase
    * Gap / Growth 1
    * Synthesis
      * DNA Replication: 8 hrs. per 3 mil. bp.
    * Gap / Growth 2
  * Mitotic Phase
    * Mitosis
    * Cytokinesis
* DNA Replication
  * Semi-Conservative: One Template/Parental Strand + One Synthesized Strand
  * Multiple Replication Origins
    * Initiator proteins to weaken H-bond between template strands
    * Result in two replication forks (Bidirectional from the origin)
    * Human DNA Polymerase: 100bp per sec (1000 bp per sec for bacteria pol.)
  * DNA Polymerase synthesized from 5' to 3'
    * Add dNTP and release 2 phosphates
    * Leading Strand: Continuous Synthesis
    * Lagging Strand: Okazaki Fragments sealed via DNA ligase
    * Multiple subtypes
      * (Type A) $\gamma$-DNA Pol. → Mitochondria (5' to 3' or 3' to 5')
      * (Type B) $\delta$-DNA Pol. → Lagging Strand (and Leading Strand)
      * (Type B) $\varepsilon$-DNA Pol. → Leading Strand
      * The rest → DNA Repairs
  * Primer required (RNA Primase) as starting point of polymerization
    * RNA Primer removed (exonuclease) and refilled with DNA (DNA Pol.)
    * Higher Error Rate → Temporary
  * Helicase: Unwind helical double-strand DNA
    * Motor protein move along phosphodiester backbone
    * Single-strand DNA-binding protein avoid closing before DNA Pol.
  * DNA Topoisomerase: Untie knots on DNA strands
    * Knots caused by helicase excessively twisting DNA
    * Type I: Cut one Strand in helical turn and rejoin without turn
    * Type II: Cut DNA strands and let another strand pass through the cut
      * Not required in DNA Replication
* Polymerase Chain Reaction (PCR): Artificial DNA Replication
  * Denature: ~ Helicase + Single-strand DNA-binding Protein + DNA Topoisomerase I
  * Annealing: ~ DNA Primase
  * Extension: ~ DNA Polymerase
* DNA Repair & Autocorrection
  * DNA Damages
    * Spontaneous reactions, Metabolic Chemicals, Replication Error
    * Drugs, Chemicals, Radiation
    * Alternative base pairs (G-T, C-A) are possible (proofread required)
  * DNA Polymerase: $1/10^{5}$ error rate (bacteria w/o proof-reading)
    * Have proof-reading: $1/10^{7}$ error rate
      * Cleave incorrect strand; replace; continue
      * Only for DNA pol. w/ exclusively 5'-to-3' direction (Identifiable template)
    * Have mismatch repair: $1/10^{9}$ error rate
  * Depurination (A,G hydrolysis; leaving backbone)
    * Lost base pair (Lost A,G as template) / Unchanged (T,C as template)
    * 5000 depurination per cell per day
  * Depyrimidination (T,C hydrolysis)
    * Similar to depurination but slower
  * Deamination (C→T)
    * Hydrolysis C → U ($\ce{N=C(NH2) ->\[+H2O\]\[-NH3\] N-C(=O)}$)
    * Protection: Uracil-DNA glycosylase (Remove U) → approx. Depyrimidination
  * DNA Repair
    * Excision: Nuclease (specialized for each DNA damage) cut damaged nucleoside (base, backbone)
    * Resynthesis: DNA Polymerase fills the gap
    * Ligation: DNA Ligase seals the backbone
  * Pyrimidine Dimer Formation
    * Factor: UV
    * 2 pairs of double bonded carbon → four-membered ring
    * Most common: Thymine dimer
  * DNA Double-strand break
    * Cannot rely on complementary strand for correction
    * Non-homologous End Joining
      * DNA Ligase joins the break
      * Potential loss of DNA fragments
    * Homologous Recombination
      * Compare with homologous chromosome
  * Somatic Mutation
    * Damage Somatic Cells
    * Affected Region
    * Normal offspring gamete
  * Germline Mutation
    * Damage Parental Gametes
    * Affect entire organism
    * Some offspring gamete are mutated

## Chromosome Regions

* Centromere
  * ~center of chromosome
  * Attach with Spindle fiber during cell division
* Telomere
  * Long-repetitive at the end of chromosome
  * Protect Ends of Linear Chromosome
  * Avoid Abnormal cell growth and proliferation
  * Lagging Strand cannot replicate until the end of the parental strand
    * Will be shorten overtime → Aging timer
    * Stop DNA replication if telomere too short → No further cell division
  * Telomerase will lengthen Telomere to a certain length
    * Extend the repetitive sequence on template strand
    * Stop the aging timer during development
    * Will be deactivated after growth → Start aging timer
      * Child only except stem cells, blood cells, liver, epithelial cells
    * Over-activation → Potential Cancer

## Reference

* Handout RAID135
